<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student_Score</class>
 <widget class="QDialog" name="Student_Score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(134, 197, 255);</string>
  </property>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1011</width>
     <height>137</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>115</width>
      <height>115</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/pic/tubiao.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>631</width>
      <height>115</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>60</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Student Score</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_teacher">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>173</width>
      <height>107</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>173</width>
      <height>107</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
	border-radius:10px;
	
background-color: rgb(255, 0, 0);
	font-family:Arial;</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>1011</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QComboBox" name="comboBox_namelist">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Student Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>20180101</string>
    </property>
   </widget>
   <zorder>comboBox_namelist</zorder>
   <zorder>label_3</zorder>
   <zorder>label_6</zorder>
   <zorder>frame_4</zorder>
   <zorder>label_5</zorder>
   <zorder>lineEdit</zorder>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>471</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>2017.9-2018.2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2018.3-2018.7</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Term:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Week:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_teacher_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;Sans Serif&quot;;
	border-radius:10px;
	
background-color: rgb(255, 0, 0);
	font-family:Arial;</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>77</y>
      <width>451</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 10pt &quot;Sans Serif&quot;;</string>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Chinese</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Math</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>English</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Physics</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Chemistry</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Biology</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>90</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>77</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>90</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>78</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>94</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>98</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>89</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>89</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>77</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>75</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>90</string>
     </property>
    </item>
    <item row="1" column="5">
     <property name="text">
      <string>97</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>78</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>87</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>89</string>
     </property>
    </item>
    <item row="2" column="5">
     <property name="text">
      <string>96</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>88</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>76</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>88</string>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string>87</string>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string>78</string>
     </property>
    </item>
    <item row="3" column="5">
     <property name="text">
      <string>94</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>94</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>80</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>88</string>
     </property>
    </item>
    <item row="4" column="3">
     <property name="text">
      <string>98</string>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string>87</string>
     </property>
    </item>
    <item row="4" column="5">
     <property name="text">
      <string>93</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>89</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>87</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>87</string>
     </property>
    </item>
    <item row="5" column="3">
     <property name="text">
      <string>94</string>
     </property>
    </item>
    <item row="5" column="4">
     <property name="text">
      <string>76</string>
     </property>
    </item>
    <item row="5" column="5">
     <property name="text">
      <string>86</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>88</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>76</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>90</string>
     </property>
    </item>
    <item row="6" column="3">
     <property name="text">
      <string>88</string>
     </property>
    </item>
    <item row="6" column="4">
     <property name="text">
      <string>88</string>
     </property>
    </item>
    <item row="6" column="5">
     <property name="text">
      <string>95</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>531</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>77</y>
      <width>64</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Term:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>77</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>2017.9-2018.2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2018.3-2018.7</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_7">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>45</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Week:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>45</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>Subject:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>23</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>~</string>
    </property>
   </widget>
   <widget class="QwtPlot" name="qwtPlot_1">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>124</y>
      <width>511</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ClosedHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);</string>
    </property>
    <property name="canvasBackground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>170</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_subject1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>95</width>
      <height>47</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Sans Serif&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Chinese</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Math</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Physics</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chemistry</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Biology</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_teacher_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255,255,255);
font: 12pt &quot;Sans Serif&quot;;
	border-radius:10px;
	
background-color: rgb(255, 0, 0);
	font-family:Arial;</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
